corehq/apps/app_manager/templates/app_manager/partials/settings/supported_languages.html

```diff
@@ -41,7 +41,7 @@
     <br/>
     <input type="checkbox" data-bind="checked: smartLangDisplay"/>
     {% trans "Smart language display in forms" %}.
-    <a href="https://confluence.dimagi.com/display/commcarepublic/Hide+languages+in+forms" target="_blank">Learn more</a>
+    <a href="https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2281275512/Language+Configuration+Translation" target="_blank">Learn more</a>
```

Comment:
This points to a table of contents page that I think the user wouldn't be happy about. Is this link what was previously linked to? https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143958174/Language+Configuration+in+CommCareHQ